docs/setup/install-existing.md

```diff
@@ -50,7 +50,19 @@ AKS. It binds Azure Active Directory identities to your Kubernetes pods. Identit
 Kubernetes pod to be able to communicate with other Azure components. In the particular case here we need authorization
 for the AGIC pod to make HTTP requests to [ARM](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview).
 
-Follow the [AAD Pod Identity installation instructions](https://github.com/Azure/aad-pod-identity#demo) to add this component to your AKS.
+To install AAD Pod Identity to your cluster:
+
+   - *RBAC enabled* AKS cluster
+
+  ```bash
+  kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/v1.5.5/deploy/infra/deployment-rbac.yaml
+  ```
+
+   - *RBAC disabled* AKS cluster
+
+  ```bash
+  kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/v1.5.5/deploy/infra/deployment.yaml
```

Comment:
["Managed Identity Operator" assignment to AKS service principal]( https://github.com/Azure/aad-pod-identity/blob/0c27b974116b911b3b56e4cdfc171d84f4d5269e/README.md)?

Reply:
In the specific example we give here, Identity is created in the node resource group were cluster sp has contributor access.